package.json

```diff
@@ -79,4 +79,4 @@
     "stylelint-prettier": "2.0.0",
     "svgo": "2.8.0"
   }
-}
+}
```

Comment:
@abhigyanghosh30 I think linter complains about this one, does your editor remove new lines on save or something?

Reply:
Oh prettier was not configured properly on my setup. It did not work on .json files